### PR TITLE
Setup ESLint config and apply to existing files

### PR DESCRIPTION
**Pull request by newmskywalker**
Setting up eslint so we can catch potential runtime errors and to apply a consistent style guide. 

This should work very nicely with the Prettier ESLint extension for VSCode. Let me know if you need help setting that up!

### CODE REVIEW

**Comment by j-mateo** [comment]
@HenryQuillin

File: src/App.js [hunk 1]
```diff
@@ -1,90 +1,67 @@
...
 const StyledBackground = styled("div")({
-  backgroundImage: `url("https://i.postimg.cc/TPTmSRWB/bg-img.png")`,
-  height: "100%",
-  position: "absolute",
-  left: 0,
-  width: "100%",
+  backgroundImage: "url(\"https://i.postimg.cc/TPTmSRWB/bg-img.png\")",
   backgroundPosition: "center",
-  backgroundRepeat: "repeat",
+  backgroundRepeat: "no-repeat",
```
Comment (j-mateo) [inline src/App.js:13]: I might have picked the wrong value while fixing merge conflicts

**Review by j-mateo — COMMENTED** [review]
Looks like I may have missed some changes while resolving merge conflicts. Will take another look to make sure this PR doesn't change any functionality

**Comment by HenryQuillin** [comment]
Thanks for this addition @j-mateo! I just made a commit before seeing this PR which is why there are more conflicts.

**Comment by j-mateo** [comment]
@HenryQuillin 

Ahh that makes sense. Let me rebase it to uptake these changes

**Comment by j-mateo** [comment]
Just fixed those conflicts. These changes shouldn't change any functionality or UI, so let me know if you see anything off!

**Comment by HenryQuillin** [comment]
Ok. I'll take a look

File: src/data/SongData.js [hunk 2]
```diff
@@ -22,14 +23,13 @@ const songData = {
...
     release_date: new Date(2017, 5).toString(),
     description: "J.Cole's best",
-    album_image:
-      "https://images.genius.com/6d0fbbc7ce189a8c81671ef92546446e.1000x1000x1.png",
+    album_image: "https://images.genius.com/6d0fbbc7ce189a8c81671ef92546446e.1000x1000x1.png",
     contributors: {
       1: { name: "John", role: "Producer", stake: 0.25 },
       2: { name: "Dan", role: "Sound engineer", stake: 0.25 },
-      3: { name: "J Cole", role: "Singer", stake: 0.5 },
+      3: { name: "J Cole", role: "Singer", stake: 0.5 }
```
Comment (j-mateo) [inline src/data/SongData.js:30]: we can add trailing commas to our eslint file if we prefer that, but it'll probably apply it to every array and object

**Comment by HenryQuillin** [comment]
Ok just fixed a tiny thing but I think its all good now. Ready to merge!

File: src/data/SongData.js (hunk 2 above)
Comment (HenryQuillin) [reply src/data/SongData.js:30]: I'm cool with whatever you prefer.

**Comment by j-mateo** [comment]
Sweet! Can you merge it? I don't have access rights yet

**Pull request merged**